src/main/OP.ts

```diff
@@ -32,7 +32,7 @@ export class OP {
 
   public constructor(opts: { builder?: OPBuilder; responseOpts?: AuthenticationResponseOpts; verifyOpts?: VerifyAuthenticationRequestOpts }) {
     this._authResponseOpts = { ...createResponseOptsFromBuilderOrExistingOpts(opts) };
-    this._verifyAuthRequestOpts = { ...createVerifyRequestOptsFromBuilderOrExistingOpts(opts) };
+    this._verifyAuthRequestOpts = { ...createVerifyRequestOptsFromBuilderOrExistingOpts(opts) } as Partial<VerifyAuthenticationRequestOpts>;
```

Comment:
What is the reason we need a partial all of a sudden. Sounds like something is off

Reply:
yes, changed some upper method calls expected a partial when it fact they have received a full object. no need for this, removed